### ISSUE TITLE
Suggestions on training on real world robot

### ISSUE THREAD

**Issue by lucasjinreal** [hume-vla/hume#10]
Hello. We aim to fine - tune the Hume pretrained model on our custom - built bimanual robots that have 14 degrees of freedom (DOF). 

Are there any specific aspects or considerations that you would suggest we should be cautious about?

Could you tell for just finetune a single task, totally new arm settings, would that work? (our arm is based on AGILex, piper arm, with two gripper)

**Comment by hume-vla** [comment]
Hello, we are honored that you are interested in our work. Sorry for the late response.

For a new robot setup, if the data meets the requirements (such as enough episodes, moderate task difficulty, and a proper collection process), then our model should be able to work properly.

For other training details, you can refer to our answer in this [issue](https://github.com/hume-vla/hume/issues/5)

**Comment by lucasjinreal** [comment]
Hi, the open-sourced code and provided pretrained model seem constrained with DoF 7. Is there much modification needed to support 14 DoF bimanual tasks?

I am mainly further concerned if it can work or not considering the pretrained model aims at DoF 7 (some dimensions are fixed with it).

**Comment by hume-vla** [comment]
Hi, don't need to worry about the DOF of the pre-trained model. The only parts of the model that require pre-trained are the VLM and action denoising head (corresponding to the weights of PI0) in System 2. As mentioned in this [issue](https://github.com/hume-vla/hume/issues/12#issuecomment-3183761762), Everything else (except DINO) is trained from scratch on the fine-tuning dataset. For the pre-trained action denoising head, the maximum supported action space dimension is 32, and it will perform padding for actions with a dimension less than 32.
